crates/db_views/src/post_view.rs

```diff
@@ -133,6 +142,7 @@ impl PostView {
         community_person_ban::all_columns.nullable(),
         post_aggregates::all_columns,
         community_follower::all_columns.nullable(),
+        community_moderator::all_columns.nullable(),
```

Comment:
You don't need to select this I don't think, in order to be able to filter on it.

Reply:
Alright, did it in [1ad9839](https://github.com/LemmyNet/lemmy/pull/3176/commits/1ad98396b7fbf3c2280268d4e4d55c2ddaed525d)